src/features/classification/security-controls/SecurityControls.js

```diff
@@ -1,62 +1,102 @@
 // @flow
 import * as React from 'react';
 import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
 
 import type { Controls, ControlsFormat } from '../flowTypes';
 
 import SecurityControlsItem from './SecurityControlsItem';
 import { getShortSecurityControlsMessage, getFullSecurityControlsMessages } from './utils';
 import { DEFAULT_MAX_APP_COUNT, SECURITY_CONTROLS_FORMAT } from '../constants';
+import SecurityControlsModal from './SecurityControlsModal';
```

Comment:
@jstoffan, is this something that will be add to the eslint rule itself? It seems like a style nit pick that I am seeing from different PR now. Would be good to make it a standard to avoid time spend during development.

Reply:
I agree that would be ideal.